rasa_core/actions/action.py

```diff
@@ -157,6 +159,18 @@ def __str__(self) -> Text:
         return "UtterAction('{}')".format(self.name())
 
 
+class ActionBack(Action):
+    """Revert the tracker state by two user utterances."""
+
+    def name(self) -> Text:
+        return ACTION_BACK_NAME
+
+    def run(self, dispatcher, tracker, domain):
+        from rasa_core.events import UserUtteranceReverted
+
+        return [UserUtteranceReverted(), UserUtteranceReverted()]
```

Comment:
ok I know i said this would be fine but i just tested it out and i feel like we need some sort of utterance or something that tells the user they've gone back. Or repeat the previous action? not 100% sure what the best UX here is -- @Ghostvv thoughts?

Reply:
We should not just replay the last action, I think that would be pretty confusing if it doesn't come with a message.
If verbosity is not a problem we could do an utterance 
"OK, I forgot the last thing you said, just like you asked.
Now, where were we?
You said: [2nd to last user input before /back]
and I said [2nd to last action]"

Reply:
I'm not sure how much we should go into this though, we shouldn't overcomplicate this. @Ghostvv any thoughts?

Reply:
these two `UserUtteranceReverted` will not cause to repeat previous action

Reply:
I suggest the same as with restart, utilize `utter_back` template:
```
        # only utter the template if it is available
        dispatcher.utter_template("utter_back", tracker,
                                  silent_fail=True)
```

Reply:
I know it's not going to cause a repetition of the previous action, I was asking whether we should do that :P

Reply:
also we decided on `/back` not being mapped to `action_back` by default right?

Reply:
no, by default I don't think we should repeat previous action, I think utterance should be enough

We could add to the docs example, that if you want to repeat previous action, return `[UserUtteranceReverted(), UserUtteranceReverted(), ActionReverted()]`

Reply:
ok cool

Reply:
@tmbo @Ghostvv do we want `/back` to be a default mapping or no?

Reply:
I don't have strong open opinion here, why not?

Reply:
idk, we decided at some point we didn't want to do that for some reason. But let's just add it anyways

Reply:
if we already have an action, why not to add intent